session-name-1/activities/activity-folder-1/sol/README.md

```diff
@@ -0,0 +1,17 @@
+# Name: Web/Binary: Session Name: Challenge Name
```

Comment:
I don't think the solution README should repeat the description in the private activity README.

I would just detail the actual solution. A structure could be:

```
# Activity Name: Solution

Description of vulnerability

Insight on the solution

Detailing of the solution, with references to solution files.
Mention if there is a certain environment where the solution is to be run.
```